### ISSUE TITLE
In Safari, the color of text in a disabled TextField is not legible

### ISSUE THREAD

**Issue by jrop** [OfficeDev/office-ui-fabric-core#1199]
In Safari (Version 13.0.2 (15608.2.30.1.1)) and `@uifabric/styling@7.4.0`, the color of text in a disabled TextField is not legible (see the screenshots below).  After playing around with the CSS, and some help from [this StackOverflow answer](https://stackoverflow.com/questions/262158/disabled-input-text-color), the following seems to fix the issue:

```
.ms-TextField.is-disabled .ms-TextField-field {
  -webkit-text-fill-color: rgb(161,159,157);
}
```

Without the above styles, the TextField appears as:
![image](https://user-images.githubusercontent.com/1404053/67812850-beb9cf80-fa65-11e9-843b-4d46fc0e3e76.png)

With the style above applied, it appears correctly:
![image](https://user-images.githubusercontent.com/1404053/67812877-cd07eb80-fa65-11e9-809d-9b786b2affaa.png)

**Comment by Jahnp** [comment]
Hi there @jrop, apologies for the delay. It looks like this issue is for https://github.com/OfficeDev/office-ui-fabric-react/. This repo is for Office UI Fabric Core, which only contains basic CSS styling. If this issue still repros for you today, please reopen it using one of [Fabric React's issue templates](https://github.com/OfficeDev/office-ui-fabric-react/issues/new/choose).

Because this issue applies to that repo, I'm going to go ahead and close this.

**Comment by jrop** [comment]
@Jahnp Thanks for the info.  It looks like this recently got fixed in [OfficeDev/office-ui-fabric-react#10808](https://github.com/OfficeDev/office-ui-fabric-react/pull/10808)
